apps/user/interfaces/user.interface.ts

```diff
@@ -1,75 +1,107 @@
-export interface UserI {
-    id?: string,
-    username?: string,
-    email?: string,
-    firstName?: string,
-    lastName?: string,
-    isEmailVerified?: boolean,
-    clientId?: string,
-    clientSecret?: string,
-    supabaseUserId?: string,
-    userOrgRoles?: object
+export  interface UInvitation {
+  orgRoles?: OrgRole[];
```

Comment:
Every parameter in an interface can not be optional. Please have mandatory parameters by removing "?"

Reply:
Resolved.